Ginger/Ginger/BusinessFlowPages_New/ActionsListViewPage.xaml.cs

```diff
@@ -183,6 +187,25 @@ private void listActions_ItemDropped(object sender, EventArgs e)
                     }
                     instance = null;
                 }
+                else if (droppedItem is ApplicationAPIModel || droppedItem is RepositoryFolder<ApplicationAPIModel>)
+                {
+                    ObservableList<ApplicationAPIModel> apiModelsList = new ObservableList<ApplicationAPIModel>();
+                    if (droppedItem is RepositoryFolder<ApplicationAPIModel>)
+                    {
+                        apiModelsList = (droppedItem as RepositoryFolder<ApplicationAPIModel>).GetFolderItems();
+                    }
+                    else
+                    {
+                        apiModelsList.Add(droppedItem as ApplicationAPIModel);
+                    }
+
+                    AddApiModelActionWizardPage APIModelWizPage = new AddApiModelActionWizardPage(mContext);
```

Comment:
changing the wizard pages from outside is not the way to do.
pass the apiModelsList in AddApiModelActionWizardPage constractor which will have defualt param for it = null, and inside the constractor if the list is not null so hide first page but all behaivor should be handled inside the wizard and not from outside

Reply:
Done!
Overloaded constructor as:
WizardBase wizard = (WizardBase)Activator.CreateInstance(t, mContext);
inside AddAction() started throwing Exception due to mismatching Wizard's Constructor.